selenium/che-selenium-test/src/test/resources/projects/ProjectWithDifferentTypeOfFiles2/src/main/java.org.eclipse.qa.examples/AppController.java

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Ok, I see now where the test above is coming from, But is this folder name even allowed in Java? If it isn't, shouldn't we reject this somehow?